index.html

```diff
@@ -5,7 +5,7 @@
     <title>
       Badging API
     </title>
-    <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class=
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
     "remove"></script>
     <script class='remove'>
       var respecConfig = {
```

Comment:
```suggestion
       var respecConfig = {
        xref: "web-platform",
```

Reply:
Ahhh! That's what I was missing. Everything I tried to get Respec to auto-link to other web specs failed (hence why I have been collecting a bunch of manual dependencies at the bottom). This makes it all work.